client/reader/stream/reader-sidebar-organizations/index.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import ReaderSidebarOrganizationsList from './list';
+
+export class ReaderSidebarOrganizations extends Component {
+	static propTypes = {
+		organizations: PropTypes.array.isRequired,
+		path: PropTypes.string.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	fetchOrganization() {
+		const { organizations, path } = this.props;
+		const slug = path.split( '/' )[ 2 ];
```

Comment:
Feels a bit fragile, especially if we change the route structure. Should we tighten it up and add some checks, just in case?